code/modules/power/lighting.dm

```diff
@@ -722,14 +792,12 @@
 	if(istype(I, /obj/item/reagent_containers/syringe))
 		var/obj/item/reagent_containers/syringe/S = I
 
-		to_chat(user, "You inject the solution into the [src].")
```

Comment:
now you won't get feedback if you inject anything other than plasma or plasma dust